pyproject.toml

```diff
@@ -53,7 +54,8 @@ markers = [
 ]
 
 testpaths = ["ckan", "ckanext"]
-addopts = "--strict-markers --pdbcls=IPython.terminal.debugger:TerminalPdb --ckan-ini=test-core.ini"
+# Set sensible defaults, cli will overide these (except for flag options
+addopts = "--strict-markers --pdbcls=IPython.terminal.debugger:TerminalPdb --ckan-ini=test-core.ini --cov=ckan --cov=ckanext --cov-branch --cov-report=term-missing --junitxml=./junit/result/junit.xml   -o junit_family=legacy --store-durations --clean-durations"
```

Comment:
`--cov=ckan --cov=ckanext` are great when you run all tests, but locally it can be irritating because output is bloated when you execute just a single test(if you forget to add --no-cov or --cov-reset, or empty --cov-report).
What do you think about moving `--cov` to workflow file. All other coverage related options can remain here, they are good defaults

Reply:
have moved the --cov items back to workflow COMMON. 

If i get some time, i'll see which options in cov can be left in if there is no -cov items listed.